tests/st/bert/bert_for_pre_pretraining.py

```diff
@@ -0,0 +1,260 @@
+# Copyright 2020-2021 Huawei Technologies Co., Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+"""
+#################pre_train bert example on zh-wiki########################
+                    python bert_for_pre_pretraining.py
+"""
+
+import os
+import argparse
+import logging
+
+from mindspore.nn.wrap.loss_scale import DynamicLossScaleUpdateCell
```

Comment:
This line should be replaced with `from tinyms.layers import DynamicLossScaleUpdateCell`.